homework_unit_11/main.js

```diff
@@ -0,0 +1,61 @@
+// Example 1
+// option 1
+const result = []
+const loopArray = (someThing, count) => {
+  for (let i = 1; i <= count; i++) {
+    result.push(someThing)
+  }
+
+  return result
+}
+
+//option 2
+const result = []
+const loopArray = (someThing, count) => {
+  for (let i = 1; i <= count; i++) {
+    result.splice(result.length, 0, someThing)
+  }
+
+  return result
+}
+
+//Example 2
+const newArr = []
+const reverseArray = array => {
+  for (let i = array.length - 1; i >= 0; i--) {
+    newArr.push(array[i])
+  }
+
+  return newArr
+}
+
+
+//Example 3
+const deleteFalsyinArray = array => {
+  const result = array.filter(function(value) {
+    return value
+  })
```

Comment:
Hoặc có thể viết ngắn hơn nếu dùng arrow function
```suggestion
  const result = array.filter(value)
```